resources/views/partials/project-header.blade.php

```diff
@@ -1,6 +1,8 @@
 <div class="project-header">
   <figure class="project-header-media">
-      @thumbnail('banner')
+    {!! get_the_post_thumbnail(SinglePccProject::rootPage($id, $post), 'banner') !!}
```

Comment:
```suggestion
    {!! get_the_post_thumbnail(SinglePccProject::rootPage(get_the_id(), $post), 'banner') !!}
```

Reply:
Hi @jpdimond, I'm still seeing a PHP warning about this — I believe it's because `$id` is undefined in the template. an you apply this suggestion?

I also added one more suggested change below to address the linting error that's being raised in the GitHub Actions job. Once those two changes are addressed I think this is ready to merge. Thanks so much!